contracts/test/vault/harvester.js

```diff
@@ -28,10 +28,37 @@ describe("Harvester", function () {
     await comp.connect(governor).transfer(compoundStrategy.address, compAmount);
   };
 
+  let fixture;
+
+  beforeEach(async function () {
+    fixture = await loadFixture(compoundVaultFixture);
```

Comment:
nit: We aren't using the fixtures the way it's supposed to be. So, using `loadFixture` has zero effect on the performance. We should switch to use `deployments.createFixture` instead. I already started doing that on the Uniswap V3 branch. It'd be nice if we can do that for all new tests as well: <https://github.com/OriginProtocol/origin-dollar/pull/1258/files#diff-42bf690234c6c24cf49e6eee681c1d54e88995355d811b66366a640e03e935dbR1253>

Related issues: https://github.com/OriginProtocol/origin-dollar/issues/1259